app/client/templates/elements/eventsTable.js

```diff
@@ -49,8 +49,9 @@ Template['elements_event_table'].helpers({
             searchQuery = TemplateVar.get('search'),
             limit = TemplateVar.get('limit'),
             collection = Events,
-            selector = this.ids ? {_id: {$in: this.ids}} : {};
-
+            selector = this.ids ? {_id: {$in: this.ids.slice(Number((limit+1)*-1))}} : {}; 
```

Comment:
It might be that you now get events in a wrong order.
As using `{limit: 10}` as an option for the cursor, will make sure you get the last 10 items, which are also sorted as you wanted.

This basically asks for the x items, without making sure they are sorted properly.

*As long as the ids we pass into are in the right order that should work.*

but im not sure if you ever tried the "show more" button, you could get funky results there.